scripts/git_clean

```diff
@@ -1,17 +1,25 @@
 #!/bin/bash
 #
-# Clean chaches, build artifacts etc
+# Clean chaches, build artifacts etc, recursively from the directory from which script is called.
```

Comment:
So this tripped me up yesterday, because git-clean "cleans the working tree by recursively removing files that are not under version control, _starting from the current directory_. [[1]]"(https://git-scm.com/docs/git-clean). 

That's why I added the comment. :)